subcommands/waves/rollout.go

```diff
@@ -74,6 +74,12 @@ A maximum number of devices rolled out using this flag cannot exceed 10000.`,
 Also accepts a filename containing a comma-separated list via "--uuids=@path/to/file.name".
 A maximum number of devices rolled out using this flag cannot exceed 10000.`,
 	)
+	rollout.Flags().BoolP("dry-run", "", false,
+		"Only show what would happen without an actual rollout. Most useful with --print-xxx flags.")
+	rollout.Flags().BoolP("print-uuids", "", false,
+		"Print the device UUIDs that were rolled out to (would be rolled out to with --dry-run).")
```

Comment:
`device UUIDs that were rolled out` - it sounds a bit odd to me. I think these are UUIDs of devices that a wave update was rolled out to.
I defer it to @kprosise .

Reply:
A full sentence is `device UUIDs that were rolled out to`... The last preposition is important ;) .

Reply:
We roll out a wave to devices... at least that's the terminology we use in all help messages, including this file.  The word "[target]" gives no added context to this.

I'll defer this to @kprosise...

The full form would be:
`Print a list of UUIDs of devices to which a wave was rolled out (would be rolled out with --dry-run)`.
But I _believe_ that within a context of the full help message this can be truncated to the present sentence, and still be grammatically and contextually correct, while I find it neater.

Reply:
`"Print the device UUIDs that wave was rolled out to..."` looks fine to me. (just three more symbols :)).